src/components/LabelBasicSelect/index.tsx

```diff
@@ -0,0 +1,32 @@
+import Label from '../styled-components/Label';
+import BasicSelect from '../styled-components/BasicSelect';
+
+interface LabelBasicSelectProps {
+  label?: string;
+  text?: string;
+  value?: string;
+  onChange?: (event: React.ChangeEvent<HTMLSelectElement>) => void;
+  options?: { value: string; label: string }[];
+  onBlur?: (event: React.ChangeEvent<HTMLSelectElement>) => void;
+  hasError?: boolean;
+  errorMessage?: string;
+}
```

Comment:
타입 속성에 모두 ?를 찍으면 타입을 선언하는데 의미가 사라질 거 같아요.
사용해야 될 타입은 ? 를 풀어주는 게 어떨까요?
```suggestion
interface LabelBasicSelectProps {
  label?: string;
  text: string;
  value: string;
  onChange: (event: React.ChangeEvent<HTMLSelectElement>) => void;
  options: { value: string; label: string }[];
  onBlur?: (event: React.ChangeEvent<HTMLSelectElement>) => void;
  hasError?: boolean;
  errorMessage?: string;
}